### ISSUE TITLE
M1 Build

### ISSUE THREAD

**Issue by emilljungberg** [spinicist/riesling#37]
Trying to build on M1 throws this error, from most recent pull on main branch

```
$./bootstrap.sh
-- Running vcpkg install
d8187afd-ea4a-4fc3-9aa4-a6782e1ed9af
CMake Error at buildtrees/0.vcpkg_dep_info.cmake:13 (message):
  base
Call Stack (most recent call first):
  buildtrees/0.vcpkg_dep_info.cmake:29 (vcpkg_triplet_file)
  buildtrees/0.vcpkg_dep_info.cmake:46 (vcpkg_get_dep_info)



-- Running vcpkg install - failed
CMake Error at cmake/vcpkg/scripts/buildsystems/vcpkg.cmake:831 (message):
  vcpkg install failed.  See logs for more information:
  /Users/emil/Code/riesling/build/vcpkg-manifest-install.log
Call Stack (most recent call first):
  cmake/toolchain.cmake:14 (include)
  /opt/homebrew/Cellar/cmake/3.22.1/share/cmake/Modules/CMakeDetermineSystem.cmake:124 (include)
  CMakeLists.txt:2 (project)


CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
CMake Error: CMAKE_CXX_COMPILER not set, after EnableLanguage
-- Configuring incomplete, errors occurred!
```

The log file `/Users/emil/Code/riesling/build/vcpkg-manifest-install.log` has the same information as on the top (0.vcpkg_dep_info etc...)

**Comment by spinicist** [comment]
Looks like I left a stray debugging `message(FATAL )` in the M1 triplet, which, er, will make CMake quit when it reaches that statement. I've removed that and pushed to `main`.

**Comment by emilljungberg** [comment]
Still the same issue I'm afraid...

**Comment by emilljungberg** [comment]
Fixed by b0e5ad4
